x-pack/plugins/core_telemetry/index.d.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export interface IVisState {
```

Comment:
I think you and I come from a C# world, where prefixing interfaces with `I` is the norm. In TypeScript, it's allowed, but generally not done. The TypeScript compiler codebase has a style guide which forbids it, and it's become a defacto rule of thumb to avoid. Just a suggestion.